lib/profiling/README.md

```diff
@@ -164,15 +164,40 @@ it should already exist. For example:
     ||psy_test:invoke_1_update_field:u||                 0.00||                 0.00||                 0.00||                    1||                44.47||                 0.00||
 ```
 
+### [TAU](./tau)
+
+This wrapper library uses the [TAU profiling library](https://www.cs.uoregon.edu/research/tau).
+Detailed linking instructions are in [``tau/README.md``](./tau/README.md).
+
+The output is written to several files called ``profile.x.y.z``, one for each process and thread.
+Simple textual output can be seen by using ``pprof``:
+
+```
+    Reading Profile files in profile.*
+
+    NODE 0;CONTEXT 0;THREAD 0:
+    ---------------------------------------------------------------------------------------
+    %Time    Exclusive    Inclusive       #Call      #Subrs  Inclusive Name
+                  msec   total msec                          usec/call
+    ---------------------------------------------------------------------------------------
+    100.0        0.343        0.375           1           2        375 .TAU application
+      8.0         0.03         0.03           1           0         30 invoke_0:r0
+      0.5        0.002        0.002           1           0          2 invoke_1_update_field:r0
+
+```
+In general is recommended to first pack the profiling output files into one file using
+``paraprof --pack``, and then using ``paraprof`` to visualise the profiling results in the GUI.
+
+
 ## Compilation
 
 The top level ``Makefile`` can be used to compile the profiling-library
 interfaces included in PSyclone. The command ``make TARGET`` where ``TARGET``
-is one of ``template``, ``simple_timing``, ``dl_timer``, ``drhook``, ``nvidia``
-or ``lfric_timer`` will only compile the corresponding library interface.
+is one of ``template``, ``simple_timing``, ``dl_timer``, ``drhook``, ``nvidia``,
+``lfric_timer``, or ``tau``, will only compile the corresponding library interface.
 
-**Note** that compilation currently does not include ``dl_timer``, ``drhook``
-and ``nvidia`` profiling-library interfaces since they require external
+**Note** that compilation currently does not include ``dl_timer``, ``drhook``,
+``tau``, and ``nvidia`` profiling-library interfaces since they require external
 libraries to be available. For the same reason, the ``all`` target
```

Comment:
As far as I can see that is already broken on trunk. I've fixed this ... and actually, now that I have rewritten and pushed this, I realised it was actually correct - you just have to continue reading the ```make all``  command in the next line and then next paragraph. Anyway, I think it's easier to read now, since obviously this was easy to misunderstand.

Reply:
Oh yes. Thanks for improving it :-)